src/scripts/collate_bad_medgen_xrefs.py

```diff
@@ -0,0 +1,45 @@
+"""Collate bad medgen xrefs
+
+From excel file containing workseets about MedGen conflicts, create a text file that lists all the bad CUIs for xrefs
+that need to be removed from Mondo.
+
+See also: https://github.com/monarch-initiative/mondo-ingest/issues/273
+
+Prerequisites:
+1. Python package 'openpyxl' to read .xlsx (left out of requirements.txt since this is a temporary script)
```

Comment:
Requires `openpyxl`, but didn't put this in `requirements.txt` as I am thinking this process is temporary considering making a final commit that removes it all.

Reply:
Nico [thinks it's fine to keep these one-time scripts / goals, etc](https://github.com/monarch-initiative/mondo/pull/6560#discussion_r1295660735), so I decided to add this to the requirements file. Maybe it's not entirely necessary, but it's only 1 dependency, with few transitive dependencies.